browser/src/map/handler/Map.KeyboardShortcuts.ts

```diff
@@ -146,10 +161,127 @@ class KeyboardShortcuts {
 
 const keyboardShortcuts = new KeyboardShortcuts();
 
+// Default shortcuts.
 keyboardShortcuts.definitions.set('default', new Array<ShortcutDescriptor>(
     // disable multi-sheet selection shortcuts in Calc
     new ShortcutDescriptor('spreadsheet', 'keydown', Mod.CTRL | Mod.SHIFT, 'PageUp', undefined, undefined),
     new ShortcutDescriptor('spreadsheet', 'keydown', Mod.CTRL | Mod.SHIFT, 'PageDown', undefined, undefined),
+
+    new ShortcutDescriptor(null, 'keydown', 0, 'F1', null, null, null, (() => {
+        app.map.showHelp('online-help-content');
```

Comment:
could you use  `dispatchAction` more? - it allows to share functionality in menubar, toolbars, notebookbar and more

it calls map.dispatch() which is defined in one of `...Toolbar.js` files

Reply:
this will help to remove `callback` and make this list clean - ofc can be done in the next PR if you want

Reply:
I removed callbacks and we have a clean docdispatcher.ts now.